resources/js/features/events/components/AchievementSet/AchievementsListItem/AchievementsListItem.tsx

```diff
@@ -101,61 +101,63 @@ export const AchievementsListItem: FC<AchievementsListItemProps> = ({
-            <span
-              title={t('Total unlocks')}
-              className={cn(
-                unlocksTotal === unlocksHardcoreTotal && unlocksHardcoreTotal > 0
-                  ? 'font-bold'
-                  : null,
-                'cursor-help',
-              )}
-            >
-              {formatNumber(unlocksTotal)}
-            </span>
-            {unlocksTotal !== unlocksHardcoreTotal ? (
-              <span className="cursor-help font-bold" title={t('Hardcore unlocks')}>
-                {'('}
-                {formatNumber(unlocksHardcoreTotal)}
-                {')'}
+            <p className="-mt-1.5 hidden text-center text-2xs md:block">
+              {t('{{percentage}} unlock rate', {
+                percentage: formatPercentage(unlockHardcorePercentage),
+              })}
+            </p>
+
+            <p className="mb-0.5 flex gap-x-1 text-2xs md:mb-0 md:justify-center md:text-center">
+              <span
+                title={t('Total unlocks')}
+                className={cn(
+                  unlocksTotal === unlocksHardcoreTotal && unlocksHardcoreTotal > 0
+                    ? 'font-bold'
+                    : null,
+                  'cursor-help',
+                )}
+              >
+                {formatNumber(unlocksTotal)}
               </span>
-            ) : null}
-            {t('of')}
-            <span title={t('Total players')} className="cursor-help">
-              {formatNumber(playersTotal)}
-            </span>
-            <span className="md:hidden">
-              {'–'} {formatPercentage(unlockHardcorePercentage)}
-            </span>
-            <span className="hidden sm:inline md:hidden">{t('unlock rate')}</span>
-          </p>
-
-          <BaseProgress
-            className="h-1"
-            max={1521}
-            segments={[
-              {
-                value: unlocksHardcoreTotal,
-                className: 'bg-gradient-to-r from-amber-500 to-[gold]',
-              },
-              {
-                value: unlocksTotal - unlocksHardcoreTotal,
-                className: 'bg-neutral-500',
-              },
-            ]}
-          />
-        </div>
+              {unlocksTotal !== unlocksHardcoreTotal ? (
+                <span className="cursor-help font-bold" title={t('Hardcore unlocks')}>
+                  {'('}
+                  {formatNumber(unlocksHardcoreTotal)}
+                  {')'}
+                </span>
+              ) : null}
+              {t('of')}
+              <span title={t('Total players')} className="cursor-help">
+                {formatNumber(playersTotal)}
+              </span>
+              <span className="md:hidden">
+                {'–'} {formatPercentage(unlockHardcorePercentage)}
+              </span>
+              <span className="hidden sm:inline md:hidden">{t('unlock rate')}</span>
+            </p>
+
+            <BaseProgress
+              className="h-1"
+              max={1521}
```

Comment:
Looks like I wrote a bug here. I bet this is also what's causing the progress bar scale issue. I think we can fix this by doing `max={playersTotal}`, since it can't be `null` at this point in the tree.